### DISCUSSION TITLE
https://theme-hope.vuejs.press/guide/advanced/extend.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#833] [Announcements]
# https://theme-hope.vuejs.press/guide/advanced/extend.html

vuepress-theme-hope supports extending just like @vuepress/theme-default. You can create your own theme based on vuepress-theme-hope and use it locally or publish it according t...

https://theme-hope.vuejs.press/guide/advanced/extend.html

**Comment by Druvidz** [comment]
The example on overriding the layout should show how to import  "theme-hope" layout (and then add the "ChangeLog" layout); instead of importing a custom made layout built out from 0. 
This basically shows the "overriding" when we are expecting an "extend"
